add limitations and migration info to router tabs docs (#38945)
expo/expo: docs/pages/router/advanced/native-tabs.mdx

## Customizing tab bar items

> > **important** Native tabs are an experimental feature available in SDK 54 and above, and their API is subject to change.

Reviewer: ```suggestion
> **important** Native tabs is an experimental feature available in SDK 54 and above, and its API is subject to change.
```

In that case, let's use correct grammar here. Native tabs are a single feature.

### Icon

> > **important** Native tabs are an experimental feature available in SDK 54 and above, and their API is subject to change.

Reviewer: ```suggestion
> **important** Native tabs is an experimental feature available in SDK 54 and above, and its API is subject to change.
```

  </NativeTabs>
```

Liquid glass on iOS automatically changes colors based on if the background color is light or dark. There is no callback for this, so you need to use a `PlatformColor` to set the color of the icon.

```tsx app/_layout.tsx
import { DynamicColorIOS } from 'react-native';
import { NativeTabs, Icon } from 'expo-router/unstable-native-tabs';

export default function TabLayout() {
  return (
    <NativeTabs
        style={{
          // For the text color
          color: DynamicColorIOS({
            dark: "white",
            light: "black",
          }),
          // For the selected icon color
          tintColor: DynamicColorIOS({
            dark: "white",
            light: "black",
          }),
        }}
      >
      <NativeTabs.Trigger name="index">
        <Icon sf="house" selectedSf="house.fill" drawable="custom_home_drawable" />
      </NativeTabs.Trigger>
      <NativeTabs.Trigger name="settings">
        <Icon
          src={require('../../../assets/setting_icon.png')}
          selectedSrc={require('../../../assets/selected_setting_icon.png')}
        />
      </NativeTabs.Trigger>
    </NativeTabs>
  )
}
```

### Label

> > **important** Native tabs are an experimental feature available in SDK 54 and above, and their API is subject to change.

Reviewer: ```suggestion
> **important** Native tabs is an experimental feature available in SDK 54 and above, and its API is subject to change.
```
